### DISCUSSION TITLE
Unable to add files on macOS ARM

### DISCUSSION THREAD

**Discussion by HamishPoole** [yermak/AudioBookConverter#585] [MacOS]
<img width="1527" alt="Screen Shot 2022-09-12 at 13 52 12" src="https://user-images.githubusercontent.com/23442241/189570819-ee2b8c4d-507c-4f3f-a32b-ae06dfb7e0d0.png">

Apologies for opening a very bad issue.

 Will document further later today.

tldr: Clicking add file does not let me add a file with the touchpad to the converter on macOS ARM does not open another dialogue box.

App fully installed successfully.

**Comment by yermak** [comment]
macOS ARMs are not currently supported. Or better say not tested.

**Comment by yermak** [comment]
Duplicates #580

**Comment by yermak** [comment]
Beta for Mac OS X is available
https://github.com/yermak/AudioBookConverter/issues/580#issuecomment-2268738667
